udata_front/theme/gouvfr/assets/js/components/discussions/threads.vue

```diff
@@ -1,15 +1,3 @@
-<!--
```

Comment:
You should probably rename this file. The storybook shows:
```
Failed to resolve import "./Threads.vue" from "udata_front/theme/gouvfr/assets/js/components/discussions/Threads.stories.js". Does the file exist?
```

Reply:
Case insensitive file system are the worst. I'm renaming it on Github.